service/rtc/server.go

```diff
@@ -105,10 +106,25 @@ func (s *Server) Start() error {
 
 	s.log.Debug("rtc: found local IPs", mlog.Any("ips", s.localIPs))
 
+	udpLocalIPs := s.cfg.ICEAddressUDP.Parse()
+	tcpLocalIPs := s.cfg.ICEAddressTCP.Parse()
+
+	// If listening on specific IPs, ensure the localIPs are filtered accordingly.
+	// We should discard anything that isn't appearing in either udpLocalIPs or tcpLocalIPs.
+	if len(udpLocalIPs) > 0 || len(tcpLocalIPs) > 0 {
+		s.localIPs = nil
+		for _, ip := range localIPs {
+			if slices.Contains(udpLocalIPs, ip.String()) || slices.Contains(tcpLocalIPs, ip.String()) {
+				s.localIPs = append(s.localIPs, ip)
+			}
+		}
+		s.log.Debug("rtc: ICE addresses set, overriding local IPs", mlog.Any("localIPs", s.localIPs))
```

Comment:
can we change to something like "rtc: ICE addresses set, filtered and only using local IPs"? (just to be a bit more clear when viewing logs)